track.py

```diff
@@ -179,8 +186,15 @@ def __init__(self, config, pathJSON='tracks'):
         self.title = data['metadata']['id']
 
         self.importSpeedLimitTuples(data['speed limits']['values'], data['speed limits']['units']['velocity'])
+
         self.importGradientTuples(data['gradients']['values'], data['gradients']['units']['slope'])
 
+        self.importCurvatureTuples(data['curvatures']['values'] if 'curvatures' in data else [(0.0, "infinity", "infinity")], 
```

Comment:
Nice. In principle, gradients are also optional, could you apply the same logic there? I forgot to do so.